tests/functional/test_sync_wrap.py

```diff
@@ -1,27 +1,27 @@
 from tests.functional.conftest import set_quota_and_rate_limit
 from tests.scripts.pds_request import PdsRecord
 import pytest
+from functools import partial
 from polling2 import poll, TimeoutException
-from pytest_bdd import scenario, given, when, then, parsers
-from .config_files.config import TEST_PATIENT_ID, PROXY_NAME, PDS_BASE_PATH
-from .utils.helper import find_item_in_dict
+from pytest_bdd import given, when, then, parsers
+import pytest_bdd
+from .configuration.config import TEST_PATIENT_ID, PDS_BASE_PATH
+from .utils.helpers import find_item_in_dict
 import json
 
+scenario = partial(pytest_bdd.scenario, './features/sync_wrap.feature')
+
 
 @pytest.mark.apmspii_832
```

Comment:
can apmspii_832 be made more descriptive? appreciate it may be out of scope for this ticket

Reply:
Probably out of the scope of this ticket, and similar to the other ticket marker, has anunknown purpose.